### PR TITLE
fix: add specific typing to helper return

### PR DESCRIPTION
**Pull request by julianiff**
- Update helper functions to return specific method name
- Bump to release new version

### CODE REVIEW

File: packages/app-bridge/package.json [hunk 1]
```diff
@@ -1,77 +1,77 @@
 {
-    "name": "@frontify/app-bridge",
-    "author": "Frontify Developers <developers@frontify.com>",
-    "version": "3.0.0-beta.98",
-    "description": "Package to establish communication between Frontify and marketplace apps",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/Frontify/brand-sdk",
-        "directory": "packages/app-bridge"
+  "name": "@frontify/app-bridge",
```
Comment (SamuelAlev) [inline packages/app-bridge/package.json:2]: Formatting is broken

Comment (SamuelAlev) [inline packages/app-bridge/package.json:2]: Formatting is broken

**Review by SamuelAlev — CHANGES_REQUESTED** [review]

**Review by marguslant0 — APPROVED** [review]
🙌

File: packages/app-bridge/package.json (hunk 1 above)
Comment (julianiff) [reply packages/app-bridge/package.json:2]: stupid linter

Comment (julianiff) [reply packages/app-bridge/package.json]: done, thx for checking

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**